packages/admin-next/dashboard/src/routes/locations/location-service-zone-shipping-option-create/components/create-shipping-options-form/create-shipping-options-prices-form.tsx

```diff
@@ -36,9 +37,12 @@ export const CreateShippingOptionsPricesForm = ({
     limit: 999,
   })
 
+  const { price_preferences: pricePreferences } = usePricePreferences({})
```

Comment:
In theory you’d might not get all price preferences here. For now I think we should add a very high limit and select only the fields we need to keep the response as small as possible.